### DISCUSSION TITLE
Voltage calibration question

### DISCUSSION THREAD

**Discussion by smurfix** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#34] [Q&A] [answered]
Hi,

The documentation currently only has a single sample values for a 9-volt transformer on a 110-Volt line. Could you please document how to modify this value for a 12-V transformer and/or a 230V line?

**Comment by CircuitSetup** [comment, marked as answer]
You will need to calibrate the AC transformer to the meter for it to read accurately. The calibration procedure is located here: https://github.com/CircuitSetup/Split-Single-Phase-Energy-Meter#voltage-procedure

If you're using ESPHome, you can view the output in the ESPHome web interface.
